### ISSUE TITLE
chroot: can't execute '/bin/sh': No such file or directory

### ISSUE THREAD

**Issue by jeffbrl** [iximiuz/cdebug#23]
Thanks for writing this software. 

When attempting to connect to caddy-docker-proxy container, I get an error. Environment is Rocky Linux 9.3 VM on ESXi server (64 bit Intel).

```
[jeffl@portainer10 ~]$ cdebug -v
cdebug version 0.0.16 (built: 2024-02-18T15:19:35Z commit: 3a7b9f64339229e0bf3ca2bf55b5e724ef851b5e)
[jeffl@portainer10 ~]$ docker run -d -v /var/run/docker.sock:/var/run/docker.sock lucaslorentz/caddy-docker-proxy:2.8.10
7b92cc877f5af3c12a8f14e249f255bd3a3534b69c074ec20b8a5f606535a072
[jeffl@portainer10 ~]$ cdebug exec -ti 7b92cc877f5
chroot: can't execute '/bin/sh': No such file or directory
[jeffl@portainer10 ~]$
```

**Comment by iximiuz** [comment]
Interesting! The `exec` command works just fine if you specify the shell explicitly:

```
$ cdebug exec -ti <container_id> sh
/ # 
/ # ps auxf
PID   USER     TIME  COMMAND
    1 0         0:00 /bin/caddy docker-proxy
   56 0         0:00 sh /.cdebug-entrypoint.sh
   65 0         0:00 sh
   66 0         0:00 ps auxf
```

It's a regression from the [recent batch of changes](https://github.com/iximiuz/cdebug/commit/1a9a533b2698265908cfd55c58aec7361f8898ef#diff-73a232efb932732b71d8ea1513c61d3fed49fec9f5ac234d21965244868c4dfdR292-R298). However, instead of reverting the change, I'll try merging #12 tomorrow - in particular, it should fix it.

**Comment by iximiuz** [comment]
Fixed in `0.0.17`. Thanks for reporting!

**Comment by jeffbrl** [comment]
Much appreciated. I really enjoy your newsletter, blog articles, and labs. Keep up the great work.
